src/matrix/net/request/replay.ts

```diff
@@ -14,29 +14,36 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import {
-    AbortError,
-    ConnectionError
-} from "../../error.js";
+import {AbortError, ConnectionError} from "../../error.js";
+import type {IRequestOptions, RequestFunction} from "../../../platform/types/types.js";
+import type {RequestResult} from "../../../platform/web/dom/request/fetch.js";
+
+interface IOptions extends IRequestOptions {
```

Comment:
Same here, see https://github.com/vector-im/hydrogen-web/blob/master/doc/TS-MIGRATION.md#use-type-rather-than-interface-for-named-parameters-and-pojo-return-values